atom/common/lib/asar.coffee

```diff
@@ -53,11 +53,20 @@ asarStatsToFsStats = (stats) ->
   }
 
 # Create a ENOENT error.
-createNotFoundError = (asarPath, filePath) ->
+createNotFoundError = (asarPath, filePath, callback) ->
```

Comment:
The method handles cases for both sync and async, and we dont throw in the async case unless the callback is not provided following the same behaviour as iojs
```
Most fs functions let you omit the callback argument. If you do, a default callback is used that rethrows errors.
```

Should it still be renamed ?